packages/user/src/components/Invitation/InvitationsTable.tsx

```diff
@@ -114,10 +114,7 @@ export const InvitationsTable = ({
                 <Tag
                   key={role + index}
                   value={role}
-                  style={{
-                    background: role === "ADMIN" ? "#6366F1" : "#22C55E",
-                    width: "5rem",
-                  }}
+                  className={role === "ADMIN" ? "primary" : "success"}
```

Comment:
you changed it here but I don't see any css change needed for it. Was it already done before? Please make sure it works as it was before.